README.md

```diff
@@ -1,140 +1,98 @@
-
-Give the new `platform` user all the admin privileges, i.e. make them a super user.
-
-You can create the `platform` user from the postgresql shell using:
-
-```
-postgres=# create user platform with superuser password 'QdYx3D5y';
-```
-
-* You also need to have [Git](https://git-scm.com/) installed in order to get the Community Platform code from [GitHub](https://github.com/coderdojo).
-
-### Mac Users
-To compile native modules you'll need to install xcode from the Mac App Store.
-
-### Docker
-If you have issues setting up, we do have a Docker setup [here](https://hub.docker.com/r/josmas/coderdojo-local-zen/) and [here](https://hub.docker.com/r/butlerx/coderdojo-local-zen/) that might be more useful
-- [josmas](https://hub.docker.com/r/josmas/coderdojo-local-zen/) container is designed for local development
-- [butlerx](https://hub.docker.com/r/butlerx/coderdojo-local-zen/) container is designed for testing and auto starts the services and runs e2e tests
+- [Docker](https://docs.docker.com/engine/installation/) Docker engine version 1.13.0 - Zen's
+development environment run completely in docker giving each mircoservice its own container
+- [Docker-compose](https://docs.docker.com/compose/install/) version 1.10.0 or higher - this is used
+to start the containers and set the env variables
+- You also need to have [Git](https://git-scm.com/) installed in order to get the Community Platform
+code from [GitHub](https://github.com/coderdojo).
 
 ## Code Setup
 
-Next step is to get the Community Platform code cloned and up and running. To do that you clone this repo and use the `localdev` command to both setup and run the code:
+Next step is to get the Community Platform code cloned and up and running. To do that you clone this
+repo and each micro service:
 
 ```
 $ git clone https://github.com/CoderDojo/cp-local-development.git && cd cp-local-development
-$ nvm use
-$ npm install
+$ git clone https://github.com/CoderDojo/cp-zen-platform.git workspace-zen/cp-zen-platform
+$ git clone https://github.com/CoderDojo/cp-events-service.git workspace-zen/cp-events-service
+$ git clone https://github.com/CoderDojo/cp-dojos-service.git workspace-zen/cp-dojos-service
+$ git clone https://github.com/CoderDojo/cp-users-service.git workspace-zen/cp-users-service
+$ git clone https://github.com/CoderDojo/cp-badges-service.git workspace-zen/cp-badges-service
+$ git clone https://github.com/CoderDojo/cp-e2e-tests workspace-zen/cp-e2e-tests
 ```
 
-Then run `node ./localdev.js` to make sure the `localdev` tool can run ok. You should see output similar to the following:
+#### Docker-compose
 
+To first set up the local development environment run
 ```
-Usage "./localdev.js <command>" where command is one of:
-  "init zen": does a fresh setup of your local dev environment
-  "run zen": runs all the services in your system
-  "testdata zen": loads test data for each service
-  "test zen": run the e2e tests
+$ docker-compose run --rm testdata
```

Comment:
What's --rm for? 
If that's the test-data db, it should always be wiped before a new test